packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -379,6 +385,13 @@ export const Tooltip = React.forwardRef(
             onMouseLeave={closeTooltip}
           >
             {text}
+            {keybindingHint && (
+              <span className={clsx(classes.keybindingHintContainer, text && classes.hasTextBefore)}>
+                <VisuallyHidden>(</VisuallyHidden>
+                <KeybindingHint keys={keybindingHint} format="condensed" variant="onEmphasis" size="small" />
+                <VisuallyHidden>)</VisuallyHidden>
+              </span>
+            )}
```

Comment:
I noticed VO announced shift twice due to the usage of ⇧. Similar behavior in NVDA, but NVDA announces the icon's description rather than shift. 

I see this was hidden via `aria-hidden`, but still announces due to the tooltip referencing it via `aria-describedby`, which nullifies the default behavior of it being ignored. Not a big blocker since it still conveys the meaning, just with added redundancy. Wonder if we'd want to address this somehow in `KeyboardingHint`.

Reply:
Is this in Chrome? [I noticed this bug](https://github.com/primer/react/pull/5252#:~:text=aria%2Dkeyshortcuts.-,aria%2Dlabel%20bug,-There%20is%20one) and I think it specifically has something to do with the `popover`. Unfortunately as it's a browser bug I was unable to find a good workaround

Reply:
Yup was chrome for VO, and Firefox for NVDA. Thanks for the context too! I'll put some time into looking into it more, but it's not a blocker for shipping this PR 🚢